daml/Marketplace/Issuance.daml

```diff
@@ -51,11 +71,47 @@ template Service
           create ReduceIssuanceRequest with ..
 
     controller provider can
+      nonconsuming Originate: [ContractId AssetDescription]
+        with
+          createOriginationCid: ContractId OriginationRequest
+        do
+          OriginationRequest{..} <- fetch createOriginationCid
+          let genAssetDescriptionVersions: C -> [C]
+              genAssetDescriptionVersions = concat . para levels
```

Comment:
It doesn't make sense to generate new versions for all types of nodes... e.g. `When today (Scale 200.0 (One usdID))` should result in a single lifecycle request, not three ... too onerous otherwise.